src/app/containers/Blocks/index.jsx

```diff
@@ -36,11 +26,14 @@ const render = blocks =>
     );
   });
 
-const MainContentContainer = ({ blocks }) => {
-  const renderedContent = render(blocks);
-  return <MainContent>{renderedContent}</MainContent>;
+Blocks.propTypes = {
+  ...mainContentModelPropTypes,
```

Comment:
This object should be renamed to be `blocksModelPropTypes`.  The `propTypes/mainContent/index.js` should also change to `propTypes/blocks/index.js`.